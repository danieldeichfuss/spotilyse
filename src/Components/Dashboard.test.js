import React from "react";
import Dashboard from "./Dashboard";
import { shallow, configure } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import ReactSixteenAdapter from "enzyme-adapter-react-16";

configure({ adapter: new ReactSixteenAdapter() });
const tracks = [{ name: "Track Name" }];
const artists = [{ artist: "Artist Name" }];
const user = { imgUrl: "img-url" };
const wrapper = shallow(
  <Dashboard tracks={tracks} artists={artists} user={user} />
);

it("renders page correctly", () => {
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
