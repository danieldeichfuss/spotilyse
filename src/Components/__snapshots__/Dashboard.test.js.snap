// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders page correctly 1`] = `
<div
  className="Dashboard"
>
  <Tabs>
    <Tab
      tabName="User"
    >
      <User
        user={
          Object {
            "imgUrl": "img-url",
          }
        }
      />
    </Tab>
    <Tab
      tabName="Favourite Song"
    >
      <FavouriteSong
        favouriteSong="Track Name"
      />
    </Tab>
    <Tab
      tabName="Top ten"
    >
      <TopTen
        artists={
          Array [
            Object {
              "artist": "Artist Name",
            },
          ]
        }
        tracks={
          Array [
            Object {
              "name": "Track Name",
            },
          ]
        }
      />
    </Tab>
    <Tab
      tabName="Discover"
    >
      <Discover
        favouriteArtist={
          Object {
            "artist": "Artist Name",
          }
        }
      />
    </Tab>
  </Tabs>
</div>
`;
